test/jdk/java/io/IO/IO.java

```diff
@@ -174,4 +185,40 @@ public void nullConsole(String method) throws Exception {
         assertEquals(1, output.getExitValue());
         output.shouldContain("Exception in thread \"main\" java.io.IOError");
     }
+
+
+    // adapted from https://raw.githubusercontent.com/junit-team/junit5/main/documentation/src/test/java/example/timing/TimingExtension.java
```

Comment:
Probably better to refer to the user guide on the web (https://junit.org/junit5/docs/snapshot/user-guide/#extensions-lifecycle-callbacks-before-after-execution)